easybuild/easyconfigs/m/mapDamage/mapDamage-2.2.1-foss-2021a-R-4.1.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'mapDamage'
+version = '2.2.1'
+versionsuffix = '-R-%(rver)s'
+
+github_account = 'ginolhac'
+
+homepage = 'https://ginolhac.github.io/mapDamage/'
+description = """mapDamage2 is a computational framework written in Python and
+    R, which tracks and quantifies DNA damage patterns among ancient DNA
+    sequencing reads generated by Next-Generation Sequencing platforms."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+checksums = ['f86cd2b1fac3e6a1d6ccc97d37ef6cf3b5be07747c3d3894472a8d1da9eb595c']
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('seqtk', '1.3'),
+    ('Pysam', '0.16.0.1'),
+    ('R', '4.1.0'),
+    ('inline', '0.3.19', versionsuffix),
+    ('RcppGSL', '0.3.10', versionsuffix),
+]
+
+use_pip = True
+sanity_pip_check = True
+download_dep_fail = True
+
```

Comment:
I tried adding `sanity_check_commands = ['%(name)s --help']` here, but that gives me a failure of:
```
== 2021-12-18 11:01:41,712 easyblock.py:3385 WARNING Sanity check: sanity check command mapDamage --help exited with code 1 (output: Seqtk executable not accessible; mapDamage has not
been intalled properly or current user does not
sufficient permissions. Expected executable at
   '/rds/bear-apps/devel/eb-sjb-up/EL8/EL8-cas/software/mapDamage/2.2.1-foss-2021a-R-4.1.0/lib/python3.9/site-packages/mapdamage/seqtk/seqtk'
)
```
So there is some hardcoding in there on where `mapDamage` expects `seqtk` to be.